src/components/Header.js

```diff
@@ -1,12 +1,102 @@
+import {useEffect, useState} from 'react';
 import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+import headerImg from './header.png';
 
 export default function Header() {
+	const [menu, setMenu] = useState(false);
```

Comment:
Perhaps it's more descriptive, but that's a minor thing 😉 
```suggestion
	const [isMenuOpen, setIsMenuOpen] = useState(false);
```

Reply:
Done.